funasr/runtime/onnxruntime/src/Audio.cpp

```diff
@@ -3,11 +3,96 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <fstream>
+#include <assert.h>
 
 #include "Audio.h"
+#include "precomp.h"
 
 using namespace std;
 
+// see http://soundfile.sapp.org/doc/WaveFormat/
+// Note: We assume little endian here
+struct WaveHeader {
```

Comment:
I suggest that you mention in some way the code is copied from
https://github.com/k2-fsa/sherpa-onnx/blob/master/sherpa-onnx/csrc/wave-reader.cc#L20
for `struct WaveHeader`.